### DISCUSSION TITLE
Is there a way to load a saved artifact from a local dir?

### DISCUSSION THREAD

**Discussion by patrick-halina** [jina-ai/finetuner#646] [General]
Finetuner has a method to download an artifact to a local directory:
https://github.com/jina-ai/finetuner/blob/a527c87745f580034476851fe093759d37318487/finetuner/hubble.py#L66

Is there a way to then load that saved artifact? The walkthroughs have a lot of documentation for saving the results of a Finetuner run, it's not clear how to load a previously saved run/artifact. From reading through the Finetuner/Hubble code, I can see that the 'artifact' abstraction is just a string to the download dir + experiment_name + '.zip'. I would be nice to just load a saved artifact from the user provided input: a download path (in the walkthroughs, you don't actually need to provide an experiment name.) Otherwise, the walkthrough or output of save_artifact() should tell the user the artifact path to remember for reloading.

**Comment by bwanglzu** [comment]
hi @patrick-halina yes, checkout [this page](https://finetuner.jina.ai/walkthrough/integrate-with-jina/#embed-documentarray)

![D366DE88-70C9-43FF-9CF5-4943283E2105](https://user-images.githubusercontent.com/9794489/210161861-88502f2a-56f0-430f-8ee6-c08193b2461e.png)

Hopefully it helps.

best

**Comment by patrick-halina** [comment]
Thanks, I missed that! It's been great playing with the Jina API so keep it up!
